addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -54,6 +59,10 @@ def _default_holiday_status_id(self):
             "\nThe status is 'To Approve', when leave request is confirmed by user." +
             "\nThe status is 'Refused', when leave request is refused by manager." +
             "\nThe status is 'Approved', when leave request is approved by manager.")
+    date_from = fields.Datetime('Start Date', readonly=True, index=True, copy=False,
+        states={'draft': [('readonly', False)], 'confirm': [('readonly', False)]}, track_visibility='onchange')
+    date_to = fields.Datetime('End Date', readonly=True, copy=False,
```

Comment:
Are those 2 fields only used in the accrual case ? Not usefull for normal leave ?